inc/cloner/class-cloner.php

```diff
@@ -2067,4 +2076,52 @@ public static function isEnabled() {
 		return (bool) $enable_cloning;
 	}
 
+	/**
+	 * Notify the source book that a clone has been completed.
+	 *
+	 * @return bool
+	 */
+	private function notifyCloneComplete(): bool {
```

Comment:
This method feels more complex than it should? Do we need to split between local/remote books? Would be possible to use Guzzle to perform the request?

Just some thoughts

Reply:
For internal requests in WP we use https://developer.wordpress.org/reference/functions/rest_do_request/. 
Guzzle would be an option, but it is unnecessary in this case. Additionally, it will keep consistency in our other requests in the cloning routine, using the native WP_Request instead Guzzle.

Thanks for the idea and suggestion, Dalcin!